mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -104,7 +149,8 @@ def mask_cross_entropy(pred,
                        reduction='mean',
                        avg_factor=None,
                        class_weight=None,
-                       ignore_index=None):
+                       ignore_index=-100,
```

Comment:
The code  `assert ignore_index is None` below conflicts with this default value.